devel/docker/Portfile

```diff
@@ -1,9 +1,9 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           github 1.0
+PortGroup           golang 1.0
 
-github.setup        moby moby 17.05.0-ce v
+go.setup            github.com/docker/cli 19.03.6 v
 revision            1
```

Comment:
The `revision` line needs to be reset to 0 when upgrading the `version`.

Reply:
Thanks @mf2k, I've missed that! Fixed.